lib/rule-context.js

```diff
@@ -70,14 +70,18 @@ var PASSTHROUGHS = [
  * @param {array} options The configuration information to be added to the rule.
  * @param {object} settings The configuration settings passed from the config file.
  * @param {object} ecmaFeatures The ecmaFeatures settings passed from the config file.
+ * @param {string} parser The parser setting passed from the config file.
  */
-function RuleContext(ruleId, eslint, severity, options, settings, ecmaFeatures) {
+function RuleContext(ruleId, eslint, severity, options, settings, ecmaFeatures, parser) {
```

Comment:
Based on the implementation, shouldn't this be `parserName` instead of `parser`?